cassandra/cqlengine/management.py

```diff
@@ -194,7 +194,7 @@ def sync_table(model):
         if table.indexes.get(index_name):
             continue
 
-        qs = ['CREATE INDEX {0}'.format(index_name)]
+        qs = ['CREATE INDEX "{0}"'.format(index_name)]
```

Comment:
Might want to use ``'CREATE INDEX {0}'.format(protect_name(index_name)`` for the extra escaping it provides.